coins.json

```
@@ -1206,6 +1206,31 @@
       "clientDocs": "https://docs.harmony.one/home/harmony-networks/harmony-network-overview/mainnet"
     }
   },
+  {
+    "id": "Oasis",
```

id should be lowercase
```suggestion
    "id": "oasis",
```